packages/styles/pagination.css

```diff
@@ -0,0 +1,77 @@
+.Pagination {
```

Comment:
For all the `px` values in this file:

* For spacing/margins please check that it exists in our [variable vars](https://github.com/dequelabs/cauldron/blob/6d33223cbca82d028aa5fcefb71d29633e021cb2/packages/styles/variables.css#L60-L70). 
* For sizing/widths/heights we should be using relative units, preferably `rem`s.